### PR TITLE
Fixes changeling HUD being added for round-start changelings.  ( Fixes #18806 )

### LINKED ISSUE #18806: Changelings produced at roundstart lack an antag hud
[Note]: # (State what the issue is from a "whats wrong" prospective. Issue reports should clearly allow maintainers to understand whats wrong and how to test/reproduce if that is not obvious. Avoid ambiguity. Start your issue report below both of these lines (or remove them))

[Admins]: # (If you are reporting a bug that occured AFTER you used varedit/admin buttons to alter an object out of normal operating conditions, please verify that you can re-create the bug without the varedit usage/admin buttons before reporting the issue.)
Mulligan and admin-produced changelings have an antag hud icon.

### PR DESCRIPTION
**Pull request by SnipeDragon**
fixes #18806

:cl: SnipeDragon
fix: Changelings created at round start now correctly receive their Antag HUD.
/:cl:

### CODE REVIEW

**Comment by SnipeDragon** [comment]
This appears to have been the only missing instance of update_changeling_icons_added from anywhere that a changeling is created.

File: code/game/gamemodes/changeling/traitor_chan.dm [hunk 1]
```diff
@@ -54,10 +54,11 @@
 
 /datum/game_mode/traitor/changeling/post_setup()
 	for(var/datum/mind/changeling in changelings)
-		changeling.current.make_changeling(changeling.current)
+		changeling.current.make_changeling()
```
Comment (lordpidey) [inline code/game/gamemodes/changeling/traitor_chan.dm]: Why this change?

**Comment by lordpidey** [comment]
Good job.  I'm the one that originally added the huds, so this was my screwup, thanks for fixing it.

**Comment by SnipeDragon** [comment]
``changeling.current.make_changeling(changeling.current) isn't needed, in fact, that was the only place in the source that make_changeling was used in that way.

File: code/game/gamemodes/changeling/traitor_chan.dm (hunk 1 above)
Comment (SnipeDragon) [inline code/game/gamemodes/changeling/traitor_chan.dm]: changeling.current.make_changeling(changeling.current) isn't needed, in fact, that was the only place in the source that make_changeling() was used in that way.  So its just a consistency update.

**Comment by MrStonedOne** [comment]
Please add `fixes #18806` to the pr up so that github auto closes that issue report when this one is merged.
